tensorflow_addons/image/median_filter_2D.py

```diff
@@ -0,0 +1,98 @@
+from tensorflow.python.ops import array_ops
+from tensorflow.python.util.tf_export import tf_export
+from tensorflow.python.ops import math_ops
```

Comment:
So unlike the PR you had in TF core... Addons only utilizes the public TF API. To do this we `import tensorflow as tf` and then access methods using say `tf.math`. Do you mind modifying your imports to match this paradigm. All the other modules should be good examples.

Reply:
_Assert3Dimage is not available in the public API. Either i need to copy the function from the source file. Or not it like i am using now. Rest tf dependencies are updated to use public API.